### PR TITLE
Translate docs/conditional-rendering.md

### PR DESCRIPTION
**Pull request by jochenboesmans**

### CODE REVIEW

File: content/docs/conditional-rendering.md [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
 permalink: docs/conditional-rendering.html
 prev: handling-events.html
 next: lists-and-keys.html
 redirect_from:
   - "tips/false-in-jsx.html"
 ---
 
-In React, you can create distinct components that encapsulate behavior you need. Then, you can render only some of them, depending on the state of your application.
+In React kan je afzonderlijke componenten maken die gedrag inkapselen dat je nodig hebt.
+Vervolgens kan je er slechts enkele van renderen, afhankelijk van de  van je applicatie.
```
Comment (Keraito) [inline content/docs/conditional-rendering.md]:
```suggestion
Vervolgens is het mogelijk om er slechts enkele van te renderen, afhankelijk van je applicatie.
```

File: content/docs/conditional-rendering.md [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
 
-Conditional rendering in React works the same way conditions work in JavaScript. Use JavaScript operators like [`if`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else) or the [conditional operator](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) to create elements representing the current state, and let React update the UI to match them.
+Conditioneel renderen in React werkt op dezelfde manier als condities werken in JavaScript.
```
Comment (Keraito) [inline content/docs/conditional-rendering.md]:
```suggestion
Conditioneel renderen in React werkt op dezelfde manier als condities in JavaScript.
```

File: content/docs/conditional-rendering.md [hunk 3]
```diff
@@ -1,18 +1,19 @@
...
+Gebruik JavaScript operatoren zoals [`if`](https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Statements/if...else) of de [conditionele operator](https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) om elementen aan te maken die de huidige  voorstellen en laat React de UI bijwerken op basis hiervan.
```
Comment (Keraito) [inline content/docs/conditional-rendering.md]:
```suggestion
Gebruik JavaScript operatoren zoals [`if`](https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Statements/if...else) of de [conditionele operator](https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) om elementen aan te maken die de huidige state voorstellen en laat React de UI bijwerken op basis hiervan.
```

File: content/docs/conditional-rendering.md [hunk 4]
```diff
@@ -42,15 +43,16 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/ZpVxNq?editors=0011)
+[**Probeer het op CodePen**](https://codepen.io/gaearon/pen/ZpVxNq?editors=0011)
 
-This example renders a different greeting depending on the value of `isLoggedIn` prop.
+Dit voorbeeld rendert een verschillende begroeting afhankelijk van de waarde van de `isLoggedIn` prop.
```
Comment (Keraito) [inline content/docs/conditional-rendering.md]:
```suggestion
Dit voorbeeld rendert een bepaalde begroeting afhankelijk van de waarde van de `isLoggedIn` prop.
```

File: content/docs/conditional-rendering.md [hunk 5]
```diff
@@ -42,15 +43,16 @@ ReactDOM.render(
...
 
-### Element Variables {#element-variables}
+### Element variabelen
```
Comment (Keraito) [inline content/docs/conditional-rendering.md]: Uppercase the `v` and don't delete the markdown id tag

File: content/docs/conditional-rendering.md [hunk 6]
```diff
@@ -116,13 +120,16 @@ ReactDOM.render(
...
-While declaring a variable and using an `if` statement is a fine way to conditionally render a component, sometimes you might want to use a shorter syntax. There are a few ways to inline conditions in JSX, explained below.
+Hoewel het verklaren van een variabele en het gebruiken van een `if` instructie een prima manier is om een component conditioneel te renderen, wil je soms een kortere syntax gebruiken.
+Er zijn een paar manieren om op dezelfde lijn condities in te voeren in JSX, zoals hieronder uitgelegd.
 
-### Inline If with Logical && Operator {#inline-if-with-logical--operator}
+### Inline If met Logische && Operator {#inline-if-with-logical--operator}
 
-You may [embed any expressions in JSX](/docs/introducing-jsx.html#embedding-expressions-in-jsx) by wrapping them in curly braces. This includes the JavaScript logical `&&` operator. It can be handy for conditionally including an element:
+Je kan [alle expressies insluiten in JSX](/docs/introducing-jsx.html#embedding-expressions-in-jsx) door ze in accolades in te sluiten.
+Dit omvat JavaScript's logische `&&`-operator.
```
Comment (Keraito) [inline content/docs/conditional-rendering.md]:
```suggestion
Hieronder valt ook de logische `&&`-operator uit Javascript.
```

File: content/docs/conditional-rendering.md [hunk 7]
```diff
@@ -116,13 +120,16 @@ ReactDOM.render(
...
+Het kan handig zijn om conditioneel een element mee te renderen:
```
Comment (Keraito) [inline content/docs/conditional-rendering.md]: Not a proper sentence

File: content/docs/conditional-rendering.md [hunk 8]
```diff
@@ -146,17 +153,18 @@ ReactDOM.render(
...
 
-It works because in JavaScript, `true && expression` always evaluates to `expression`, and `false && expression` always evaluates to `false`.
+Dit werkt omdat in JavaScript `true && expression` altijd evalueert naar `expression` en `false && expression` altijd evalueert naar` false`.
 
-Therefore, if the condition is `true`, the element right after `&&` will appear in the output. If it is `false`, React will ignore and skip it.
+Hierdoor verschijnt, als de conditie `true` is, het element meteen na `&&` in de uitvoer.
+Als ze `false` is, zal React ze negeren en overslaan.
 
-### Inline If-Else with Conditional Operator {#inline-if-else-with-conditional-operator}
+### Inline if-else met conditionele operator {# inline-if-else-with-conditional-operator}
```
Comment (Keraito) [inline content/docs/conditional-rendering.md]: why is everything lowercases and is the markdown id broken again?

**Review by Keraito — CHANGES_REQUESTED** [review]
A lot of "slordigheidsfouten", both in the translations as well as the markup. Please try to proofread your work and check your changes while you're working.

File: content/docs/conditional-rendering.md (hunk 7 above)
Comment (jochenboesmans) [reply content/docs/conditional-rendering.md]: @Keraito It seems like a proper sentence to me. Could you point to the grammatical error(s) or suggest an alternative translation?

Comment (Keraito) [reply content/docs/conditional-rendering.md]: In hindsight, my feedback was kind of useless and your sentence was not as "inproper" as I read at the time of review, apologies. I think that it's mainly hard to read as a sentence, even when I know what you're trying to say with it.

"Het kan" is too associated with the generic meaning representing "It is possible", which makes the current sentence really weird. I would suggest to actively reference the previous sentence with "Deze" or using "Hiermee" in the sentence.

If you want to stay close to the original content, I will suggest something like `Hiermee kan het handig zijn om een element conditioneel te renderen` or `Deze kan handig zijn om een element conditioneel mee te renderen`. Or if you want to be slightly more explicit: `Het kan handig zijn om deze te gebruiken voor het conditioneel renderen van een element`.

File: content/docs/conditional-rendering.md [hunk 9]
```diff
@@ -42,15 +43,16 @@ ReactDOM.render(
...
+[**Probeer het op CodePen**](https://codepen.io/gaearon/pen/ZpVxNq?editors=0011)
 
-This example renders a different greeting depending on the value of `isLoggedIn` prop.
+Dit voorbeeld rendert een bepaalde begroeting afhankelijk van de waarde van de `isLoggedIn` prop.
 
-### Element Variables {#element-variables}
+### Element Variabelen {#element-variables}
 
-You can use variables to store elements. This can help you conditionally render a part of the component while the rest of the output doesn't change.
+Je kan variabelen gebruiken om elementen op te slaan.
```
Comment (ronderksen) [inline content/docs/conditional-rendering.md]: I know both are valid, but I prefer "je kunt" over " je kan". I looked it up and it's one of those Flemish vs Dutch things, where in Flemish "je kan" is considered the more polite version and in Dutch that honor goes to "je kunt". But as I said: both are perfectly valid. The only thing is that we need consistency, so if everyone is in favor of "je kan", then we should all use that.

File: content/docs/conditional-rendering.md [hunk 10]
```diff
@@ -70,9 +72,11 @@ function LogoutButton(props) {
 }
 ```
 
-In the example below, we will create a [stateful component](/docs/state-and-lifecycle.html#adding-local-state-to-a-class) called `LoginControl`.
+In het onderstaande voorbeeld maken we een [stateful component](/docs/state-and-lifecycle.html#adding-local-state-to-a-class) genaamd `LoginControl`.
+
+Het zal ofwel `<LoginButton />` ofwel `<LogoutButton />` renderen, afhankelijk van de huidige state.
```
Comment (ronderksen) [inline content/docs/conditional-rendering.md]: "de component" is a male noun in Dutch, so: 
```suggestion
Hij zal ofwel `<LoginButton />` ofwel `<LogoutButton />` renderen, afhankelijk van de huidige state.
```

File: content/docs/conditional-rendering.md [hunk 11]
```diff
@@ -70,9 +72,11 @@ function LogoutButton(props) {
...
+Het zal ook een `<Greeting />`, uit het vorige voorbeeld, renderen:
```
Comment (ronderksen) [inline content/docs/conditional-rendering.md]:
```suggestion
Hij zal ook een `<Greeting />`, uit het vorige voorbeeld, renderen:
```

File: content/docs/conditional-rendering.md [hunk 12]
```diff
@@ -116,13 +120,16 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/QKzAgB?editors=0010)
+[**Probeer het op CodePen**](https://codepen.io/gaearon/pen/QKzAgB?editors=0010)
```
Comment (ronderksen) [inline content/docs/conditional-rendering.md]: Interesting inconsistency in the English docs: in other pages, they don't use the `**` and so far we haven't used them either. So maybe to stay consistent in the Dutch translation, don't use `**` for these links?

File: content/docs/conditional-rendering.md (hunk 7 above)
Comment (ronderksen) [reply content/docs/conditional-rendering.md]: I would even make the second sentence a subordinate clause (bijzin) of the first sentence:

```Hieronder valt ook de logische `&&`-operator uit JavaScript, waarmee je handig een element conditioneel kunt renderen.```

I think that we are used to having longer sentences in Dutch than in English (but I'm not certain it's true).

**Review by ronderksen — CHANGES_REQUESTED** [review]

File: content/docs/conditional-rendering.md (hunk 12 above)
Comment (jochenboesmans) [reply content/docs/conditional-rendering.md]: @ronderksen Yes, the early articles in the docs use empty links like `[](codepen.io/link)`. However, I personally think the fat links `[**Try it on CodePen**](codepen.io/link)`go better with the overall layout of the page. I would be in favor of adapting earlier articles to the latter format.

File: content/docs/conditional-rendering.md (hunk 7 above)
Comment (jochenboesmans) [reply content/docs/conditional-rendering.md]: Adapted to the following for now:
`
Hieronder valt ook de logische `&&`-operator uit JavaScript, waarmee je gemakkelijk een element conditioneel kunt renderen.
`

**Review by Keraito — APPROVED** [review]

**Review by ronderksen — APPROVED** [review]

**Pull request merged**